users/aaarisova/exercise/hw-11/hw-11.ipynb

```diff
@@ -0,0 +1,1846 @@
+     ]
+    }
+   ],
+   "source": [
+    "# I \n",
+    "\n",
+    "def combination_no_duplicates():\n",
+    "    combination = []\n",
+    "    for i1 in range(ord('a'), ord('z') + 1):\n",
+    "        for i2 in range(ord('a'), ord('z') + 1):\n",
+    "            letters = chr(i1) + chr(i2)\n",
+    "            if chr(i1) != chr(i2):\n",
+    "                combination.append(letters) \n",
+    "    return combination \n",
+    "\n",
+    "my = combination_no_duplicates()\n",
+    "print(my)\n",
```

Comment:
def combination_no_duplicates():
    for i1 in range(ord('a'), ord('z') + 1):
        for i2 in range(ord('a'), ord('z') + 1):
            letters = chr(i1) + chr(i2)
            if chr(i1) != chr(i2):
                print(letters, end=" ") 
        print("") 

my = combination_no_duplicates()
print(my)